### PR TITLE
Pagination: fix slot

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.
#1190

### CODE REVIEW

File: packages/pagination/src/pagination.js [hunk 1]
```diff
@@ -87,6 +87,15 @@ export default {
   },
 
   components: {
+    MySlot: {
+      render(h) {
+        return (
+          this.$parent.$slots.default
+          ? this.$parent.$slots.default[0]
+          : ''
+        );
```
Comment (QingWei-Li) [inline packages/pagination/src/pagination.js]: 那你直接 return this.$parent.$slots.default 就好了呢

Comment (Leopoldthecoder) [inline packages/pagination/src/pagination.js]: 不可以呢，`this.$parent.$slots.default` 是个数组呢，会报 `Multiple root nodes` 的错呢

**Comment by baiyaaaaa** [comment]
只能显示在sizes 和 total 的 slot么，感觉用法很死啊- -

**Comment by QingWei-Li** [comment]
@baiyaaaaa 通过 layout 自己设置各组件的位置的

**Pull request merged**